src/java.base/share/classes/sun/nio/ch/SocketChannelImpl.java

```diff
@@ -58,6 +58,7 @@
 import static java.net.StandardProtocolFamily.INET;
 import static java.net.StandardProtocolFamily.INET6;
 import static java.net.StandardProtocolFamily.UNIX;
+import static sun.nio.ch.Streams.MAX_BUFFER_SIZE;
```

Comment:
This is not generally useful for the SocketChannel implementation, it is instead for the sole use of the methods that take an input/output stream. So I think better to use Streams.MAX_BUFFER_SIZE in the two methods rather than static import.